### DISCUSSION TITLE
Is it possible to have an icon with no text on the left?

### DISCUSSION THREAD

**Discussion by edent** [badges/shields#8193] [Q&A] [answered]
I would like to create a badge which has an icon on the left, but not text.  

The nearest I can get is this:
<img src="https://img.shields.io/badge/.-Icon%20Only-ff0?logo=appveyor&style=for-the-badge" />

`https://img.shields.io/badge/.-Icon%20Only-ff0?logo=appveyor&style=for-the-badge`

I've tried adding a space `https://img.shields.io/badge/%20-Icon%20Only-ff0?logo=appveyor&style=for-the-badge` - but that removes the left hand side entirely:

<img src="https://img.shields.io/badge/%20-Icon%20Only-ff0?logo=appveyor&style=for-the-badge" />

Using the [Mongolian Vowel Separator](https://graphemica.com/180E) *sort of* works

`https://img.shields.io/badge/%E1%A0%8E-Icon%20Only-ff0?logo=appveyor&style=for-the-badge`
<img src="https://img.shields.io/badge/%E1%A0%8E-Icon%20Only-ff0?logo=appveyor&style=for-the-badge" />
But it leaves a large gap.

Is there a way to have `[icon] | [Text]` ?

**Comment by chris48s** [comment, marked as answer]
If you pass no label text but specify the `labelColor` param that will do the job

https://img.shields.io/badge/Icon%20Only-ff0?logo=appveyor&style=for-the-badge&labelColor=555

![](https://img.shields.io/badge/Icon%20Only-ff0?logo=appveyor&style=for-the-badge&labelColor=555)

No Mongolian vowel separator required :relieved:
